src/braket/pennylane_plugin/ops.py

```diff
@@ -364,6 +364,18 @@ class XY(Operation):
     par_domain = "R"
     grad_method = "A"
 
+    @staticmethod
+    def decomposition(phi, wires):
+        # XX * YY
+        return [
+            qml.CY(wires=wires),

```

Comment:
Why is this better than the decomposition before?

Reply:
We don't currently support CRX, so it would inevitably get decomposed with 2 CNOTs. This reduces the total number of 2q gates by 1.

Reply:
What about H(0) CY(0, 1) RY(0, theta) RX(1, -theta) CY(0, 1) H(0)